src/Jackett.Common/Definitions/pixelados.yml

```diff
@@ -81,12 +81,15 @@ search:
     page: 1
 
   keywordsfilters:
+    - name: diacritics
+      args: replace
+  # strip season and ep because it's added by Query
```

Comment:
Ah no.
In my examples, since I don't have access to pixelados, I will use blutopia since its also a unit3d site that I do have access to.

You are assuming that all the TV queries use the torznab t=tvsearch
Sonarr query
http://127.0.0.1:9117/api/v2.0/indexers/blutopia-api/results/torznab/api?t=tvsearch&cat=100001,100002,100003,100005&extended=1&apikey=redacted&offset=0&limit=100&q=Younger&season=1&ep=1
Jackett builds 
https://blutopia.xyz/api/torrents/filter?api_token=redacted&name=Younger+S01E01&seasonNumber=1&episodeNumber=1&categories[]=1&categories[]=2&categories[]=3&categories[]=5&sortField=created_at&sortDirection=desc&perPage=100&page=1

However, the indexer also has to support the general torznab t=search query
general torznab search
http://127.0.0.1:9117/api/v2.0/indexers/blutopia-api/results/torznab/api?t=search&extended=1&apikey=redacted&offset=0&limit=100&q=Younger+S01E01
Jackett builds
https://blutopia.xyz/api/torrents/filter?api_token=redacted&name=Younger+S01E01&sortField=created_at&sortDirection=desc&perPage=100&page=1

to wipe the season and ep for tvsearch you are going to need to code a conditional using the `.Query.Type` variable which will contain the search type: `tvsearch` or `search` or `movie` or `music` or `book`

Unfortunately, at this time, the yml processor does not support the use of variable replacements in the keywordsfilters block, which would allow for example something like:
```yaml
    - name: re_replace
      args: ["(?i)S0?(\\d{1,2})E(\\d{1,2})", "{{ if eq .Query.Type \"tvsearch\" }}{{ else }}$1x$2{{ end }}"]
```

I will open an enhancement ticket to request that feature, but until then you are going to have to revert your changes to the  keywordsfilters block.

Reply:
Great, I was looking for some similar but I can't make it work. It's much better with your customization.